modules/caddyhttp/reverseproxy/reverseproxy.go

```diff
@@ -630,7 +630,7 @@ func (h Handler) prepareRequest(req *http.Request, repl *caddy.Replacer) (*http.
 	if err != nil {
 		// OK; probably didn't have a port
 		addr, err := netip.ParseAddr(address)
-		if err != nil {
+		if err != nil { //nolint:revive
 			// Doesn't seem like a valid ip address at all
 		} else {
```

Comment:
@mholt did you really want to not return the `err` here in case of an invalid ip?

Looks like the linter might have caught an issue.